src/Doctrine/TablePrefix.php

```diff
@@ -22,7 +22,7 @@ public function getSubscribedEvents () {
 	public function loadClassMetadata ( LoadClassMetadataEventArgs $eventArgs ) {
 		$classMetadata = $eventArgs->getClassMetadata();
 
-		if ( $classMetadata->getTableName() === $this->container->getParameter( 'doctrine_migrations.table_name' ) ) {
+		if ( $classMetadata->getTableName() === 'migration_versions'/*$this->container->getParameter( 'doctrine_migrations.storage.table_storage.table_name' )*/ ) {
```

Comment:
You should use `ParameterBagInterface` instead of `ContainerInterface` as its deprecated since Symfony 5.1 : https://symfony.com/blog/new-in-symfony-4-1-getting-container-parameters-as-a-service